### ISSUE TITLE
ldap-checker module not detecting LDAPS channel binding

### ISSUE THREAD

**Issue by ghecko** [Pennyw0rth/NetExec#122]
**Describe the bug**
When I use the `ldap-checker` module, I'm getting an error when attempt to binds to LDAPS.

**To Reproduce**
Steps to reproduce the behavior i.e.:
Command: `nxc ldap XX.YY.ZZ.ZZ --port 636 -u 'USER' -p pass.txt -d 'test.local' -M ldap-checker`
Resulted in:
```
SMB         XX.YY.ZZ.ZZ     445    DCTEST1         [*] Windows 10.0 Build 17763 x64 (name:DCTEST1) (domain:test.local) (signing:True) (SMBv1:False)
LDAP        XX.YY.ZZ.ZZ     636    DCTEST1         [+] test.local\USER:XXXXXXXX 
LDAP-CHE... XX.YY.ZZ.ZZ     636    DCTEST1         LDAP Signing NOT Enforced!
LDAP-CHE... XX.YY.ZZ.ZZ     636    DCTEST1         [-] ERROR while connecting to test.local: an integer is required (got type NoneType)
```

**Expected behavior**
LDAPS bindings to check status of channel binding.

**NetExec info**
 - OS: Exegol on Manjaro host
 - Version of nxc: 1.1.0
 - Installed from: pipx installation

**Comment by Marshall-Hallenbeck** [comment]
@ghecko This modules works for me in my lab, is it potentially an issue on the test host you are using?
I do agree that the error being thrown is not helpful though, so I'll look into better handling that.

![image](https://github.com/Pennyw0rth/NetExec/assets/1518719/b0318a2c-be9b-4a6c-a1b9-b454fbbe060c)

**Comment by NeffIsBack** [comment]
I can't reproduce the error but the module is not working anymore on my side. Regardless on how i set up channel binding it always says `"NEVER"`. The original project tracks it correctly tho.

<img width="539" alt="image" src="https://github.com/Pennyw0rth/NetExec/assets/61382599/8cf0a2f4-1885-490f-8457-589e8b05deb7">
<img width="657" alt="image" src="https://github.com/Pennyw0rth/NetExec/assets/61382599/80ec43a5-84a1-4190-82b8-a2dd293789a2">

**Comment by NeffIsBack** [comment]
This is fixed with #216
